.gitignore

```diff
@@ -69,46 +87,40 @@ PCbuild/arm64/
 PCbuild/obj/
 PCbuild/win32/
 Tools/unicode/data/
-.purify
-__pycache__
-autom4te.cache
-build/
-buildno
-config.cache
-config.log
-config.status
-config.status.lineno
-core
-!Tools/msi/core/
-db_home
-.hg/
-.idea/
-ipch/
-libpython*.a
-libpython*.so*
-libpython*.dylib
-libpython*.dll
-platform
-pybuilddir.txt
+/autom4te.cache
+/build/
+/buildno
+/config.cache
+/config.log
+/config.status
+/config.status.lineno
+/db_home
+/.hg/
+/ipch/
+/libpython*.a
+/libpython*.so*
+/libpython*.dylib
+/libpython*.dll
+/platform
+/pybuilddir.txt
 /pyconfig.h
-python-config
-python-config.py
-python.bat
-python.exe
-python-gdb.py
-python.exe-gdb.py
-reflog.txt
-.svn/
-tags
-TAGS
-.coverage
-coverage/
-externals/
-htmlcov/
+/python-config
+/python-config.py
+/python.bat
+/python.exe
```

Comment:
We could globally ignore `*.exe`, with a specific exception for `!Lib/distutils/command/*.exe`.